generate_testdata.sh

```diff
@@ -59,31 +59,47 @@ scaffold_test_project() {
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false
     $kb create api --group crew --version v1 --kind Captain --controller=true --resource=true --make=false --force
     $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation
+    if [ $project == "project-v3" ]; then
+      $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation --force
+    fi
+
     $kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
     $kb create webhook --group crew --version v1 --kind FirstMate --conversion
-    $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
-    $kb create webhook --group crew --version v1 --kind Admiral --defaulting
-    $kb create api --group crew --version v1 --kind Laker --controller=true --resource=false --make=false
+
     if [ $project == "project-v3" ]; then
-      $kb create webhook --group crew --version v1 --kind Captain --defaulting --programmatic-validation --force
+      $kb create api --group crew --version v1 --kind Admiral --plural=admirales --controller=true --resource=true --namespaced=false --make=false
+      $kb create webhook --group crew --version v1 --kind Admiral --plural=admirales --defaulting
+    else
+      $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
+      $kb create webhook --group crew --version v1 --kind Admiral --defaulting
     fi
```

Comment:
checked that the changes here does not change the current scenarios and just add a new one to check a plural with an irregular form for an api and webhook